content/docs/intro/console/accounts-and-organizations/accounts.md

```diff
@@ -19,9 +19,9 @@ Your account lets you authenticate into the Pulumi Console, where you can do the
 following:
 
 * Manage your profile settings, including your account password, access tokens, and subscriptions
-* [Add an organization]({{< relref "organizations" >}}) backed by Atlassian, GitHub, GitLab, or a SAML
+* [Add an organization]({{< relref "/docs/intro/console/accounts-and-organizations/organizations/" >}}) backed by Atlassian, GitHub, GitLab, or a SAML
```

Comment:
I suspect it's the trailing slashes in some of these that's causing the check failures here. (Running `make serve` or `make build` locally _should_ surface these pretty quickly.)